plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationContext.java

```diff
@@ -85,6 +91,14 @@ public void setUnknown(Unknown unknown) {
         this.unknown = unknown;
     }
 
+    public String getMergeStrategy() {
+        return mergeStrategy;
+    }
+
+    public void setMergeStrategy(String mergeStrategy) {
+        this.mergeStrategy = mergeStrategy;
```

Comment:
no test coverage for this? (should be covered when the merge strategy is set to non default

Reply:
I'm wondering if this method is useful. I get the strategy name from ENV instead of a setter method.

Reply:
if you don't need it then remove it, can be added later.